config/core/test-configs.yaml

```diff
@@ -1860,6 +1866,13 @@ test_configs:
     test_plans:
       - baseline
 
+  - device_type: at91sam9g20ek
+    test_plans:
+      - baseline
+      - baseline-nfs
+    filters:
+      - passlist: {defconfig: ['at91_dt_defconfig']}
```

Comment:
Building this extra defconfig on staging now to then get a set of tests scheduled:
https://bot.staging.kernelci.org/job/kernel-build/57083/consoleText